### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by GitMensch**
For now providing a summary in half text / half markdown form (and already containing `-z` option).

Thoughts?

Are you fine with moving to markdown and/or have a second README (both forms)?

### CODE REVIEW

**Comment by ostash** [comment]
Feel free to convert readme to markdown completely.

File: README [hunk 1]
```diff
@@ -1,18 +1,58 @@
...
-Dependencies: elfutils (https://fedorahosted.org/elfutils/)
-
-Building:
-- create site.mak file and set FLAGS variable with paths to elfutils header and libraries
-  For example:
-
-FLAGS=-I/usr/local/elfutils/include -L/usr/local/elfutils/lib -O2 -march=native -Wl,-rpath /usr/local/elfutils/lib
-
-- make it
+Because of its own simplified format containing only the data nacessary for creating the callgrind profile the resulting fiel is commonly much smaller.
```
Comment (ostash) [inline README:6]: s/fiel/file/

File: README [hunk 2]
```diff
@@ -1,18 +1,58 @@
...
 - collect samples using 'pgcollect'
 - convert collected samples into 'callgrind' file using 'pgconvert'
 - open resulting 'callgrind' file in KCachegrind
+
+# `pgcollect` - collect samples
+Usage: `pgcollect outfile.pgdata [-z] [-F freq] [-s] {-p pid | [--] cmd}`
+
+Options to specify output:
+- `outfile.pgdata` name of output file
+- `-z` enable compression
```
Comment (ostash) [inline README:20]: I'd prefer this to be added after implementation.

Comment (GitMensch) [reply README:20]: Sure, that's one of the reasons for the draft.

When you like the general approach I'll finish the conversion to markdown and fix typos, then force-push without the -z.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+- convert collected samples into 'callgrind' file using `pgconvert`
+- open resulting 'callgrind' file in KCachegrind
+
+## `pgcollect` - collect samples
+Usage: `pgcollect filename.pgdata [-F freq] [-s] {-p pid | [--] cmd}`
+
+Options to specify output:
+- `filename.pgdata` name of output file
+
+Options to adjust profiling
```
Comment (ostash) [inline README.md:22]: Add a ':' at the end of line

File: README.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+- `-F` profile at this frequency
+- `-s` profile using software events
+
+Options to specify target
+- `-p pid` profile running process with PID=_pid_
+- `cmd` command to profile, prefix with `--` to stop command line parsing
+
+## `pgconvert` - convert
```
Comment (ostash) [inline README.md:30]:
```suggestion
## `pgconvert` - convert collected samples to `callgrind` format
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
+# perfgrind
+
+This is 'perfgrind', tools for collecting samples from Linux performance events
```
Comment (ostash) [inline README.md:3]:
```suggestion
This is `perfgrind`, tools for collecting samples from Linux performance events
```

File: README.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
+# perfgrind
```
Comment (ostash) [inline README.md:1]:
```suggestion
# `perfgrind`
```

File: README.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+
+This is 'perfgrind', tools for collecting samples from Linux performance events
+subsystem and converting profiling data to 'callgrind' format.
```
Comment (ostash) [inline README.md:4]:
```suggestion
subsystem and converting profiling data to `callgrind` format.
```

File: README.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+
+Because of its own simplified format containing only the data necessary for creating the callgrind profile the resulting file is commonly much smaller.
+
+
+# Usage
+
+## Overview
+- collect samples using `pgcollect`
+- convert collected samples into 'callgrind' file using `pgconvert`
+- open resulting 'callgrind' file in KCachegrind
```
Comment (ostash) [inline README.md:14]:
```suggestion
- open resulting `callgrind` file in KCachegrind
```

File: README.md [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+Usage: `pgcollect filename.pgdata [-F freq] [-s] {-p pid | [--] cmd}`
+
+Options to specify output:
+- `filename.pgdata` name of output file
+
+Options to adjust profiling
+- `-F` profile at this frequency
+- `-s` profile using software events
+
+Options to specify target
```
Comment (ostash) [inline README.md:26]:
```suggestion
Options to specify target:
```

File: README.md (hunk 6 above)
Comment (GitMensch) [reply README.md:1]: That's a headline, not code.

File: README.md (hunk 7 above)
Comment (GitMensch) [reply README.md:4]: Again: neither code nor something to enter. We could move from ' to " here if you like

File: README.md (hunk 8 above)
Comment (GitMensch) [reply README.md:14]: as above

**Comment by GitMensch** [comment]
I'll combine the changes and add ":" at end of non-headlines as suggested this evening.
Should we change ' to " ?
Anything else?

**Comment by GitMensch** [comment]
I'll add the license note to the docs, this should also allow changing to "GitHub compatible" GPL license for later.

**Review by ostash — APPROVED** [review]

**Comment by GitMensch** [comment]
Guess that's finished, going to the Makefile next.

**Pull request merged**